### DISCUSSION TITLE
[lazer] Game crashes while changing into the Borderless Window Type

### DISCUSSION THREAD

**Discussion by PolskiKapsel** [ppy/osu#20275] [Ideas]
Problem details:
Osu!lazer instantly crashes while changing from any other Window Type into Borderless. I managed to fix the game by quickly using Alt+Enter key combination, however the Error is still easily reproduceable, at least on my PC.

Video showing the problem: https://youtu.be/oY8Aniz92dg

osu! version: 2022.911.0-lazer

[logs.zip](https://github.com/ppy/osu/files/9552402/logs.zip)

**Comment by marcostudios** [comment]
Make sure to upload the log files, you can find them by reading https://github.com/ppy/osu/discussions/15188

**Reply by PolskiKapsel** [reply]
Done

**Comment by frenzibyte** [comment]
https://github.com/ppy/osu/issues/20276. Next time please consider either opening an issue thread or opening a discussion thread. There's no need to open both at once.

**Comment by peppy** [comment]
Probably drivers or some overlay. Closing issue as there's nothing useful in logs.
